tests/test_sanctions.py

```diff
@@ -200,6 +200,26 @@ def test__notify_non_authorizer(self, mock_send):
         args, kwargs = mock_send.call_args
         assert_true(self.user.username in args)
 
+    @mock.patch('website.mails.send_mail')
+    def test_ask(self, mock_send):
+        group = [self.user]
+        for i in range(5):
+            u = factories.UserFactory()
+            group.append(u)
+        self.sanction.ask(group)
+        authorizer = group.pop(0)
+        mock_send.assert_any_call(
+            authorizer,
+            self.sanction.AUTHORIZER_NOTIFY_EMAIL_TEMPLATE,
+            **{}
```

Comment:
@sloria  No, but I wanted to document that the function might take kwargs
```mock_send.assert_any_call(
    authorizer,
     self.sanction.AUTHORIZER_NOTIFY_EMAIL_TEMPLATE
)```
passes too